media/static-bundles.json

```diff
@@ -1,129 +1,87 @@
 {
   "css": [
     {
-      "files": [
```

Comment:
@nathan-barrett are the formatting changes here intended? I'm not sure our Prettier config is set up to include json files, so I was just wondering how this got formatted. It's not a big problem, but it could be a bit painful if you need to rebase this branch again before it merges?

If this was caused by your code editor, it might be worth double checking that it uses the project config in bedrock?

Reply:
Ah, good catch. I'll double check